### ISSUE TITLE
missing alt text on images

### ISSUE THREAD

**Issue by ZooeyMiller** [fac-13/Tonnah#13]
https://github.com/fac-13/Tonnah/blob/c657af9d802b61fbc9742d584caf281c2b0425b4/src/components/board/singlephoto/singlephoto.js#L6

plz use `alt` text 🙏
